app/schema.graphql

```diff
@@ -654,8 +664,12 @@ enum SpanKind {
   unknown
 }
 
+"""
+The sort key and direction for span connections. Must specify one and only one of either `col` or `eval_result_key`.
+"""
 input SpanSort {
-  col: SpanColumn!
+  col: SpanColumn = null
+  evalResultKey: EvalResultKey = null
```

Comment:
`oneOf` for input types is [not](https://github.com/strawberry-graphql/strawberry/issues/2421) available in strawberry